web/exemple/blog/index.html

```diff
@@ -7,6 +7,7 @@
         <meta name="description" content="Un exemplu de pagină HTML5.">
         <meta name="keywords" content="exemplu, html5">
         <title>Câteva cuvinte despre HTML 5</title>
+        <link rel="stylesheet" type="text/css" href="styles.css"></link>
```

Comment:
Nu trebuie sa modifici un fisier in acest director. Trebuie respectata structura prezentat la laborator.
La acest exercitiu trebuie sa copii exemplul acesta in directorul cu numele tau din `web/solutii/<nume_prenume>`

Pentru mai multe informatii acceseaza [pagina cursului](https://teaching.alexcoman.com/resurse/tutorial/git/2017/03/11/git-propunerea-unei-solutii/).